x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/rolemapping/TransportGetRoleMappingsAction.java

```diff
@@ -51,8 +51,9 @@ protected void doExecute(Task task, final GetRoleMappingsRequest request, final
         } else {
             names = new HashSet<>(Arrays.asList(request.getNames()));
         }
+        // TODO make sure result is deterministic
```

Comment:
What I mean here is ordered the same each time: the respond now contains the output of `ClusterStateRoleMapper#getMappings` which is a set and therefore not well-ordered.